### PR TITLE
chore(rename): org scope package

### PR DESCRIPTION
**Pull request by ericdeansanchez**
The purpose of this PR is to start the renaming process. `dist/` files have been
included to make the subsequent steps a little cleaner (less noise for the next
reviewal).

If/when this PR gets a thumbs up, I'll start the next steps:

- do repo/project renaming
- release beta
- use beta to ensure the readme is accurate and updated
- then do the changelog edits

Lmk if you think there's a better strategy/order of steps.

### CODE REVIEW

File: src/index.mjs [hunk 1]
```diff
@@ -34,7 +34,7 @@ export default class ImgixClient {
     }
 
     if (this.settings.includeLibraryParam) {
-      this.settings.libraryParam = 'js-' + VERSION;
+      this.settings.libraryParam = 'js-core-' + VERSION;
```
Comment (sherwinski) [inline src/index.mjs:37]: Is this necessary if the version number is staying on track with what we currently have?

**Review by sherwinski — APPROVED** [review]
LGTM 👍 

I noticed this and was wondering if git was feeling ok:

Screen Shot 2021-02-22 at 5 04 47 PM

Screen Shot 2021-02-22 at 5 05 12 PM

**Comment by frederickfogerty** [comment]
I'm still going to review this at a later stage but my 2c is that it'd be really nice to fix up the renames that @sherwinski mentioned so that we keep git history for the future.

File: src/index.mjs (hunk 1 above)
Comment (ericdeansanchez) [reply src/index.mjs:37]: No it's not necessary, but I had wondered what you thought about it. I'm good with keeping just the `js-` portion.

File: assert_version.cjs [hunk 2]
```diff
@@ -6,7 +6,7 @@ var packageVersion = require('./package').version;
     return 0;
   } else {
     process.stdout.write(
-      'FAIL: package.json and lib/imgix-core-js.js versions do not match!\n',
+      'FAIL: package.json and @imgix/js-core versions do not match!\n',
```
Comment (frederickfogerty) [inline assert_version.cjs:9]:
```suggestion
      'FAIL: package.json and src/constants.mjs versions do not match!\n',
```

File: dist/imgix-js-core.d.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for imgix-core-js.js
+// Type definitions for @imgix/js-core.js
```
Comment (frederickfogerty) [inline dist/imgix-js-core.d.ts:1]:
```suggestion
// Type definitions for imgix-js-core.*.js
```

I would also accept just `@imgix/js-core` without the `.js`

**Review by frederickfogerty — CHANGES_REQUESTED** [review]
Some places I found `imgix-core-js` still existing:

- `.prettierignore`
- `CHANGELOG.md` (you said you were going to handle this later so that's fine)
- `package-lock.json` (since package.json has changed we need to change this)
- `README.md`
- `.github` folder (PR template, etc)

Also, if we could somehow ensure the git renames work as expected to keep history that would be amazing. Sometimes it is not possible and needs to be done with two commits/PRs.

File: dist/imgix-js-core.d.ts (hunk 3 above)
Comment (ericdeansanchez) [reply dist/imgix-js-core.d.ts:1]: Nice catch!!!

**Comment by ericdeansanchez** [comment]
TODO:
- [ ] `.prettierignore`
- [ ] `CHANGELOG.md` (you said you were going to handle this later so that's fine)
- [x] `package-lock.json` (since package.json has changed we need to change this)
- [ ] `README.md`
- [ ] `.github` folder (PR template, etc)

**Comment by ericdeansanchez** [comment]
@frederickfogerty we split the commits but the diff on GitHub still displays the same weirdness that @sherwinski posted above. Locally, git displays the correct thing (both pre and post rebasing/re-managing the commits).

**Pull request merged**